redash/handlers/users.py

```diff
@@ -27,7 +27,7 @@ def get(self):
     def post(self):
         req = request.get_json(force=True)
         require_fields(req, ('name', 'email'))
-
+        req['email'] = req['email'].lower()
```

Comment:
Shouldn't we transform email to lower case everywhere where we receive it as input?